src/ui/dialogs/spring_pane.rb

```diff
@@ -337,6 +260,13 @@ def mounted_users_excitement
     excitement
   end
 
+  def update_mounted_users
+    simulate
+  end
+
+  def update_mounted_users_excitement
+  end
```

Comment:
We should either refactor that out completlty, or somehow annotate that as a TODO.
Right now to fix this maybe it is enough to just print somethink like: currently this does nothing, just do warn us that this functionality is complelty out (also feel free to add me on a trello card to refactor it out)